launcher/tests/unit/UI/LangDialog.spec.js

```diff
@@ -0,0 +1,16 @@
+import { mount } from '@vue/test-utils'
+import LangDialog from "../../../src/components/UI/LangDialog";
+
+
+describe('LangButton.vue',()=>{
```

Comment:
`describe()` is used to group several related tests. You can also use it do make it easier to tell your tests apart if they have similar names.
You used `describe('LangButton.vue',()=>{` in **LangDialog.spec.js** you might want to change it to 
`describe('LangDialog.vue',()=>{`.

Reply:
Thank you for being careful.